app/boards/shields/dumbpad/dumbpad.conf

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Uncomment to turn on logging, and set ZMK logging to debug output
+# CONFIG_ZMK_USB_LOGGING=y
+# CONFIG_SENSOR_LOG_LEVEL_DBG=y
```

Comment:
Just curious, what did you need this specific debug logging for?

Reply:
That was leftover from some debugging. I'll remove.